internal/cluster/monitor.go

```diff
@@ -0,0 +1,71 @@
+package cluster
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/go-openapi/swag"
+	"github.com/openshift/assisted-service/pkg/requestid"
+
+	"github.com/jinzhu/gorm"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+func (m *Manager) ClusterMonitoring() {
```

Comment:
if u are already changing it maybe we can add workers here and run in parallel? It looks like it will very long run when we will have 100 clusters for example.
I think we need to redesign it a little and work in parallel. 
@ronniel1 @filanov

Reply:
@ronniel1 @filanov to say the truth it looks like this monitor will work very hard and will be not easy to finish it.
it has loop in loop and can go through very large list. 
Not sure it is worth to aggregate 2 monitors to one.

Reply:
it's a good idea, but i suggest to do it in a different ticket